tests/NPSP.robot

```diff
@@ -280,5 +290,17 @@ Select Frame With Title
     
 Scroll Page To Location
     [Arguments]    ${x_location}    ${y_location}
-    Execute JavaScript    window.scrollTo(${x_location},${y_location}) 
-    
\ No newline at end of file
+    Execute JavaScript    window.scrollTo(${x_location},${y_location})
+
+Open NPSP Settings
+    [Arguments]    ${topmenu}    ${submenu}
+    Open App Launcher
+    Populate Address    Search apps or items...    NPSP Settings
+    Select App Launcher Link  NPSP Settings
+    Wait For Locator    frame    Nonprofit Success Pack Settings
+    Select Frame With Title    Nonprofit Success Pack Settings
+    Wait for Locator    npsp_settings.side_panel
+    Click Link    link=${topmenu}
+    Sleep    1
+    Click Link    link=${submenu}
+    Sleep    1
```

Comment:
It would be nice to wait for specific elements rather than sleeping here, but it's okay for now.

Reply:
I had found that sometimes clicking the links didn't work, so I found the sleep used elsewhere and added it here.

Reply:
Clicking the links triggers a render that happens asynchronously so we do need to wait, but it's always better to wait for something specific to occur (like with the Wait Until Page Contains Element or Wait For Locator keywords) rather than a Sleep, which is either going to be longer than necessary and slow the tests down, or shorter than necessary on a slow system and make the test flaky. The existing sleeps should be seen as temporary hacks...I'm okay with doing the same here for now as long as you understand why it's not ideal.

Reply:
Understood. Thanks for the clarification.